hubblestack/extmods/modules/hubble.py

```diff
@@ -28,8 +29,8 @@
 
 try:
     from nova_loader import NovaLazyLoader
```

Comment:
Shouldn't we remove this file altogether, and start using hubble.loader in hubble.py

Reply:
Yes. I was tempted to do this, but decided to mark it XXX and leave it for later. I think you're right though. Let me figure that out quick today.